libs/application/templates/accident-notification/src/lib/messages/applicantInformation.ts

```diff
@@ -12,6 +12,11 @@ export const applicantInformation = {
       defaultMessage: 'Upplýsingar um þig',
       description: 'Information about you',
     },
+    titleForReviewer: {
```

Comment:
This already exists in the file in line 61 forTheThirdParty

Reply:
Fixed!